README.md

```diff
@@ -1,17 +1,53 @@
 # Cartography
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg?style=flat-square)](LICENSE)![Version](https://img.shields.io/nuget/v/Cartography.DynamicMap?style=flat-square)![Downloads](https://img.shields.io/nuget/dt/Cartography.DynamicMap?style=flat-square)   
 
-Nventive solution for mobile app map.
+Cartography provides you 3 map related libraries for IOS, Android and windows using native map of each platform.
```

Comment:
Always use proper casing for products.
- It's **iOS**, not IOS.
- It's **Windows**, not windows.